sdk/servicebus/azure-servicebus/azure/servicebus/_common/message.py

```diff
@@ -722,32 +674,36 @@ def _add(self, add_message, parent_span=None):
                     self.max_size_in_bytes
                 )
             )
-
-        self.message._body_gen.append(message)  # pylint: disable=protected-access
+        add_batch(self._message, message.raw_amqp_message._to_outgoing_amqp_message())  # pylint: disable=protected-access
         self._size = size_after_add
         self._count += 1
         self._messages.append(message)
 
     @property
-    def max_size_in_bytes(self):
-        # type: () -> int
+    def message(self) -> LegacyBatchMessage:
+        if not self._uamqp_message:
+            raise Exception("Attempting to use legacy batch")
```

Comment:
just checking that this line will be removed after testing?

Reply:
Absolutely - sorry this was my local debugging and I committed by accident >_<